git_creation_and_tracking/git_creation_and_tracking.md

```diff
@@ -0,0 +1,780 @@
+$ mkdir moons    # make a subdirectory planets/moons
+$ cd moons       # go into moons subdirectory
+$ git init       # make the moons subdirectory a Git repository
+$ ls -a          # ensure the .git subdirectory is present indicating we have created a new Git repository
+```
+
+Is the `git init` command, run inside the `moons` subdirectory, required for
+tracking files stored in the `moons` subdirectory?
+
+[( )] Yes, running `git init` inside the `moons` subdirectory is necessary.
+[( )] It is not necessary to run `git init` inside `moons` but it can't hurt.
+[(X)] No, `moons` was already tracked as a subdirectory of `planets` and running `git init` inside `moons` could interfere with version control.
+***
+<div class = "answer">
+No. Dracula does not need to make the `moons` subdirectory a Git repository
+because the `planets` repository will track all files, sub-directories, and
+subdirectory files under the `planets` directory.  Thus, in order to track
+all information about moons, Dracula only needed to add the `moons` subdirectory
+to the `planets` directory.
+
+Additionally, Git repositories can interfere with each other if they are "nested":
+the outer repository will try to version-control
+the inner repository. Therefore, it's best to create each new Git
+repository in a separate directory. To be sure that there is no conflicting
+repository in the directory, check the output of `git status`. If it looks
+like the following, you are good to go to create a new repository as shown
+above:
+
+```console
+$ git status
+
+fatal:
+Not a git repository (or any of the parent directories):
+.git
+```
+ </div>
+ ***
+
+### Fixing a nested `init`
+
+<div class = "help">
+What if you forgot to check `git status` and accidentally created a repository nested in another repository?
+You can fix this by removing the `.git` file from the inner repository. Take another look at Dracula's code from our last example:
+
+```console
+$ cd ~/Desktop   # return to Desktop directory
+$ cd planets     # go into planets directory, which is already a Git repository
+$ ls -a          # ensure the .git subdirectory is still present in the planets directory
+$ mkdir moons    # make a subdirectory planets/moons
+$ cd moons       # go into moons subdirectory
+$ git init       # make the moons subdirectory a Git repository
+$ ls -a          # ensure the .git subdirectory is present indicating we have created a new Git repository
+```
+
+Dracula has created two tracking files, `planets/.git` which he should keep, and `planets/moons/.git` which should be removed.
+Dracula can do this by running `pwd` which will show the **present working directory** and getting the output:
+
+```console
+/Users/vlad/Desktop/planets
+```
+before running:
+
+```console
+$ rm -rf moons/.git
+```
+</div>
+
+## Tracking changes
+
+ It might seem like *making* changes and *documenting* them are two different things, but the entire point of version control is to document every change you make.
```

Comment:
```suggestion
 It might seem like **making** changes and **documenting** them are two different things, but the entire point of version control is to document every change you make.
```